addons/initialise-sprite-position/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Customizable new sprite position",
-  "description": "Change the default x/y position of new sprites.",
+  "description": "Changes the default X/Y positions of new sprites.",
```

Comment:
Each sprite only has one position, which is defined by two numbers for the X and Y axes.

Reply:
Either "default X/Y coordinates" or "default position".